### ISSUE TITLE
low argument in randint

### ISSUE THREAD

**Issue by louisabraham** [hyperopt/hyperopt#536]
Currently, randint only takes 1 argument, the upper bound.

Also, I think it should be made consistent with https://docs.python.org/3/library/random.html#random.randint and include the upper bound.

**Comment by louisabraham** [comment]
Some people use instead `hp.choice(range(low, high))`, I think it yields different results in practice.

**Comment by Mwuschnig** [comment]
I also think a lower bound would be great for randint.

**Comment by louisabraham** [comment]
@Mwuschnig I recommend https://github.com/optuna/optuna

**Comment by marctorrellas** [comment]
my understanding is that this is defined here (stocastic.py), @maxpumperla correct me if I'm wrong

```
@implicit_stochastic
@scope.define
def randint(upper, rng=None, size=()):
    # this is tricky because numpy doesn't support
    # upper being a list of len size[0]
    if isinstance(upper, (list, tuple)):
        if isinstance(size, int):
            assert len(upper) == size
            return np.asarray([rng.randint(uu) for uu in upper])
        elif len(size) == 1:
            assert len(upper) == size[0]
            return np.asarray([rng.randint(uu) for uu in upper])
    return rng.randint(upper, size=size)
```

would any of you @louisabraham @Mwuschnig  like to contribute to extend this to have an extra param lower with default (I guess) equals 0?

**Comment by maxpumperla** [comment]
implemented in #595
